pulp_docker/app/serializers.py

```diff
@@ -13,6 +13,111 @@
 from . import models
 
 
+class ManifestListTagSerializer(platform.ContentSerializer):
+    """
+    Serializer for ManifestListTags.
+    """
+
+    name = serializers.CharField(help_text="Tag name")
+    manifest_list = platform.DetailRelatedField(
+        many=False,
+        help_text="Manifest List that is tagged",
+        view_name='docker-manifest-lists-detail',
+        queryset=models.ImageManifest.objects.all()
```

Comment:
typo? s/models.ImageManifest/models.ManifestList

Reply:
Oh wow, nice catch. Surprisingly, it seems to make no difference in the output.